scripts/ci-tool.py

```diff
@@ -20,6 +20,7 @@
 import fetchai_code_quality
 
 BUILD_TYPES = ('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel')
+LOG_LEVELS = ('trace', 'debug', 'info', 'warn', 'error', 'critical', 'none')
```

Comment:
This seems unrelated, I would suggest you put it in a sperate PR along with the suggested changes for MapSerializer

Reply:
Sure.
One problem with separate PRs is that often you need these changes straight away, not once a separate PR is merged.

Reply:
https://github.com/fetchai/ledger/pull/2057